doc/developers/maintainer.rst

```diff
@@ -120,57 +118,34 @@ Making a release
    candidate period, the latest stable is two versions behind the master
    branch, instead of one.
 
-3. At this point all relevant PRs should have been merged into the `0.99.X`
-   branch. Create the source tarball:
-
-   - Wipe clean your repo::
-
-       $ git clean -xfd
-
-   - Generate the tarball::
-
-       $ python setup.py sdist
-
-   - You can also test a binary dist build using::
-
-       $ python setup.py bdist_wheel
-
-   - You can test if PyPi is going to accept the package using::
-
-       $ twine check dist/*
-
-   You can run ``twine check`` after step 5 (fetching artifacts) as well.
-
-   The result should be in the `dist/` folder. We will upload it later
-   with the wheels. Check that you can install it in a new virtualenv and
-   that the tests pass.
-
-4. Proceed with caution. Ideally, tags should be created when you're almost
+3. Proceed with caution. Ideally, tags should be created when you're almost
    certain that the release is ready, since adding a tag to the main repo can
-   trigger certain automated processes. You can test upload the ``sdist`` to
-   ``test.pypi.org``, and test the next step by setting ``BUILD_COMMIT`` to the
-   branch name (``0.99.X`` for instance) in a PR to the wheel building repo.
+   trigger certain automated processes. You can create a PR in the main repo
+   and trigger the wheel builder with the ``[cd build]`` commit marker. Then,
+   you can upload the wheels and ``sdist`` to ``test.pypi.org`` for testing.
    Once all works, you can proceed with tagging. Create the tag and push it (if
    it's an RC, it can be ``0.xxrc1`` for instance)::
 
     $ git tag -a 0.99  # in the 0.99.X branch
 
     $ git push git@github.com:scikit-learn/scikit-learn.git 0.99
 
-5. Update the dependency versions and set ``BUILD_COMMIT`` variable to the
-   release tag at:
-
-   https://github.com/MacPython/scikit-learn-wheels
+.. note::
+  
+  Before building the wheels, make sure that the ``pyproject.toml`` file is
+  up to date and using the oldest version of ``numpy`` for each Python version
+  to avoid ABI incompatibility issues. Moreover, a new line have to be included
+  in the ``pyproject.toml`` file for each new supported version of Python.
 
-   Once the CI has completed successfully, collect the generated binary wheel
+4. Once the CD has completed successfully, collect the generated binary wheel
    packages and upload them to PyPI by running the following commands in the
    scikit-learn source folder (checked out at the release tag)::
 
        $ rm -r dist # only if there's anything other than the sdist tar.gz there
        $ pip install -U wheelhouse_uploader twine
        $ python setup.py fetch_artifacts
```

Comment:
This command will download all the binary packages accumulated in a staging area on the anaconda.org hosting service and put then in your local `./dist` folder:

https://anaconda.org/scikit-learn-wheels-staging/scikit-learn/files